specs/am/architecture-management-vocab.ttl

```diff
@@ -19,18 +19,21 @@
 @prefix vann:        <http://purl.org/vocab/vann/> .
 
 @prefix oslc_am:    <http://open-services.net/ns/am#> .
-@prefix jazz_am:    <http://jazz.net/ns/dm/linktypes#> .
 
 # The ontology
 
 oslc_am:
-    a                    owl:Ontology ;
-    rdfs:label           "Architecture Management(AM)" ;
-    dcterms:description  "All vocabulary URIs defined in the OSLC Architecture Management (AM) namespace."^^rdf:XMLLiteral ;
-    dcterms:title        "The OSLC Architecture Management(AM) Vocabulary" ;
-    vann:preferredNamespacePrefix  "oslc_am" ;
-    # This needs to refer to the published source of the vocabulary, in the OASIS document library
-    dcterms:source        <https://raw.githack.com/oslc-op/oslc-specs/master/specs/am/architecture-management-vocab.ttl> .
+  a owl:Ontology ;
+  dcterms:title "The OSLC Architecture Management(AM) Vocabulary" ;
+  rdfs:label "The OSLC Architecture Management(AM) Vocabulary" ;
+  dcterms:description "All vocabulary URIs defined in the OSLC Architecture Management (AM) namespace."^^rdf:XMLLiteral ;
+  vann:preferredNamespacePrefix "oslc_am" ;
+  dcterms:publisher <https://open-services.net/about/> ;
+  dcterms:issued "2020-xx-xx"^^<http://www.w3.org/2001/XMLSchema#date> ;
```

Comment:
Should be commented out until publication

Reply:
done